src/pkg/backup/details/groups.go

```diff
@@ -116,6 +116,12 @@ func (i *GroupsInfo) uniqueLocation(baseLoc *path.Builder) (*uniqueLoc, error) {
 		if len(i.DriveID) == 0 {
 			return nil, clues.New("empty drive ID")
 		}
+	case GroupsChannelMessage:
+		category = path.ChannelMessagesCategory
+
+		if len(i.ChannelID) == 0 {
+			return nil, clues.New("empty channel ID")
+		}
 	}
 
 	loc, err := NewGroupsLocationIDer(category, i.DriveID, baseLoc.Elements()...)
```

Comment:
`DriveID` will be blank in case of channel. Which is properly handled in `NewGroupsLocationIDer`. But as discussed earlier do we still need `TeamID` info here?

Reply:
Whoops.  I this should be the channelID.  Thanks for the catch!